### ISSUE TITLE
Disable ok-to-test so the none of the commits from external contributors will trigger the tests automatically

### ISSUE THREAD

**Issue by aszecowka** [kyma-project/test-infra#50]
Currently, when external dev creates PR, there is a requirement that someone from kyma contributors adds comment 'ok-to-test', and only after that tests are executed.
Unfortunately, when a developer commits new changes, 'ok-to-test' comment is not reverted and tests will be executed immediatelly. This can leads to security issues when commits after 'ok-to-test' comment introduce malicious code.

**Comment by suleymanakbas91** [comment]
Since this is related to how Prow trigger plugin works, I created this issue in kubernetes/test-infra: https://github.com/kubernetes/test-infra/issues/10118.

**Comment by suleymanakbas91** [comment]
As a result of the issue, Prow developers decided to make `/ok-to-test` configurable by [this PR](https://github.com/kubernetes/test-infra/pull/10137). If it is disabled, only option to trigger the tests for an external contributor's PR will be commenting `/test all` to the PR.

**Comment by aszecowka** [comment]
1. AI: Disable ok-to-test
2. Find a way how to notify kyma developers that they have to execute `test all` on PR from external contributor

**Comment by suleymanakbas91** [comment]
Created this follow-up task for documentation: https://github.com/kyma-project/test-infra/issues/337
